src/platforms/android/common/dependencies.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Managing Dependencies
+sidebar_order: 1000
+---
+
+## Managing versions for multiple Sentry dependencies
+
+When multiple Sentry dependencies are used, to avoid specifying the version of each dependency, you can use a *BOM* or *Bill Of Materials*.
+
+Using Maven you can add the following to `dependencyManagement` section in your `pom.xml`:
+
+```xml {tabTitle:Maven}{filename:pom.xml}
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.sentry</groupId>
+            <artifactId>sentry-bom</artifactId>
+            <version>{{ packages.version('sentry.java', '5.1.0') }}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
+
+and then use dependencies without specifying a version, for example:
+
+```xml {tabTitle:Maven}{filename:pom.xml}
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry-android</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry-android-fragment</artifactId>
+</dependency>
+```
+
+Using Gradle 5.0 or higher, you can add the following to the `dependencies` section in your `build.gradle`:
```

Comment:
Given @bruno-garcia's comment above, do you want this content to move up on the page? (prior to the Maven content?)

Reply:
Removed maven part completely from android.